fr/orm/table-objects.rst

```diff
@@ -150,6 +151,42 @@ Liste des Events
 * ``Model.afterDelete``
 * ``Model.afterDeleteCommit``
 
+initialize
+----------
+
+.. php:method:: initialize(Event $event, ArrayObject $data, ArrayObject $options)
+
+L'event ``Model.initialize`` est déclenché après que les méthodes de
+constructeur et initialize sont appelées. Les classes table n'écoutent pas cet
```

Comment:
I'd uppercase the "table" word since it's an object name, and between backticks.